zircon.core/common/src/main/kotlin/org/hexworks/zircon/internal/component/renderer/VerticalSliderRenderer.kt

```diff
@@ -0,0 +1,35 @@
+package org.hexworks.zircon.internal.component.renderer
+
+import org.hexworks.zircon.api.Positions
+import org.hexworks.zircon.api.component.Slider
+import org.hexworks.zircon.api.component.data.ComponentState
+import org.hexworks.zircon.api.component.renderer.ComponentRenderContext
+import org.hexworks.zircon.api.component.renderer.ComponentRenderer
+import org.hexworks.zircon.api.data.Tile
+import org.hexworks.zircon.api.graphics.Symbols
+import org.hexworks.zircon.api.graphics.impl.SubTileGraphics
+
+class VerticalSliderRenderer : ComponentRenderer<Slider> {
+
+    override fun render(tileGraphics: SubTileGraphics, context: ComponentRenderContext<Slider>) {
+        val style = context.componentStyle.currentStyle()
+        tileGraphics.applyStyle(style)
+
+        val defaultStyleSet = context.componentStyle.fetchStyleFor(ComponentState.DEFAULT)
+        val invertedDefaultStyleSet = defaultStyleSet
```

Comment:
I really like this inverted concept! 👍

Reply:
I totally stole that from the ProgressBar renderer